### ISSUE TITLE
adapt pydantic.v1.error_wrappers.ValidationError: 1 validation error for Prompt

### ISSUE THREAD

**Issue by runwean** [explodinggradients/ragas#1089]
I want to customize the adapt function,  change the language to Chinese

this is code：
```py
cache_dir = ".cache"

adapt(
    metrics=evaluate_stk,
    language="Chinese",
    cache_dir=cache_dir,
    llm=llm
)
```

and this is the error
```
Traceback (most recent call last):
  File "/Users/Desktop/side2side-eval/langchain_idealab/ragas_evaluate.py", line 87, in <module>
    adapt(
  File "/Users/opt/anaconda3/envs/self-rag/lib/python3.9/site-packages/ragas/adaptation.py", line 36, in adapt
    metric.adapt(language, cache_dir=cache_dir)
  File "/Users/opt/anaconda3/envs/self-rag/lib/python3.9/site-packages/ragas/metrics/_answer_relevance.py", line 173, in adapt
    self.question_generation = self.question_generation.adapt(
  File "/Users/opt/anaconda3/envs/self-rag/lib/python3.9/site-packages/ragas/llms/prompt.py", line 181, in adapt
    return self._load(language, self.name, cache_dir)
  File "/Users/opt/anaconda3/envs/self-rag/lib/python3.9/site-packages/ragas/llms/prompt.py", line 281, in _load
    return cls(**json.load(open(path)))
  File "/Users/opt/anaconda3/envs/self-rag/lib/python3.9/site-packages/pydantic/v1/main.py", line 341, in __init__
    raise validation_error
pydantic.v1.error_wrappers.ValidationError: 1 validation error for Prompt
__root__
  output in example 1 is not in valid json format: Expecting value: line 1 column 1 (char 0) (type=value_error)
```

**Comment by Mervyn-1** [comment]
same issue

**Comment by jjmachan** [comment]
@Mervyn-1 @runwean sorry about this but do keep an eye on #774. will update that issue when we have a fix

**Comment by huangxuyh** [comment]
I've encountered the same issue. It would be greatly appreciated if the author could look into it. Thank you so much!

**Comment by Xiyuche** [comment]
It might not be an issue with the LLM API. I tried GPT-4o, GPT-4o-mini, and DeepSeek, and their translated cache JSON is almost the same, and I can't find any errors. Also, the first time you execute the adaptation and test generation, it works fine, so I guess something is wrong when it's trying to read the promt JSON saved as cache.

**Comment by stilkin-pxl** [comment]
> I guess something is wrong when it's trying to read the prompt JSON saved as cache.

Assuming this is correct, how would I clear / remove the JSON saved as cache?

**Comment by Z-oo883** [comment]
> 我遇到了同样的问题。如果作者能研究一下，我们将不胜感激。非常感谢！

请问你解决了吗

**Comment by jjmachan** [comment]
This has been fixed with v0.2 - I know finally 😅 🎉

do checkout the docs here: https://docs.ragas.io/en/stable/howtos/customizations/metrics/_metrics_language_adaptation/
reference here: https://docs.ragas.io/en/stable/references/prompt/#ragas.prompt.PromptMixin

and if you're migrating from v0.1 check out the migration docs here: https://docs.ragas.io/en/stable/howtos/migrations/migrate_from_v01_to_v02
